samples/config_helper_vulkan.cc

```diff
@@ -646,8 +646,8 @@ amber::Result ConfigHelperVulkan::CreateVulkanInstance(
     required_extensions.push_back(kExtensionForValidationLayer);
   }
 
+  available_instance_extensions_ = GetAvailableInstanceExtensions();
```

Comment:
Instead of changing this so `available_instance_extensions_ = GetAvailableInstanceExtensions();` always executes, I could just do `available_instance_extensions_ = GetAvailableInstanceExtensions();` at the start of `DumpPhysicalDeviceInfo`